main/main.go

```diff
@@ -165,6 +165,7 @@ func main() {
 		ManifestClient:  clusterinfo.NewClusterInfoClient(mgr.GetClient()),
 		OstreeClient:    ostreeClient,
 		Ops:             op,
+		PrepConfig:      &controllers.PrepConfig{Active: false, Cancel: nil},
```

Comment:
nit: Seems like `Success` is not explicitly initialized here, but the default value is false if it's not initialized.

Reply:
`Success` is explicitly initialized to false in `handlePrep` when a new reconciliation for prep-stage commences. I was avoiding to bloat the parameter initialization here, but I can add it here.